functions/src/index.ts

```diff
@@ -30,6 +63,17 @@ async function getThingspeakKeysFromPurpleAir(
   return new PurpleAirResponse(purpleAirApiResponse);
 }
 
+/**
+ * Increment the buffer index, circling back to 0 if the index is at the last point
+ * @param index - current index
+ * @param length - length of the buffer array
+ */
+const incrementIndex = (index: number, length: number) => {
```

Comment:
I'm not sure I can think of something better, but this function name makes me feel like it is going to modify the number, not just return the number to use

Reply:
N/A, we got rid of this function